src/monitor/monitor-client.c

```diff
@@ -132,7 +132,13 @@ static int ProcessMonitor(jack_nframes_t nframes, void *arg)
         goto muted;
 
     const float volume = mon->volume;
+    float smooth_volume = mon->smooth_volume;
     const bool apply_compressor = mon->apply_compressor;
+
+    //if changing gains from 0dB
```

Comment:
cant really make sense of this code comment..
the code is clear enough IMO, the comment is not needed.